tests/__snapshots__/build.test.ts.snap

```diff
@@ -14,7 +14,7 @@ exports[`gen route types 1`] = `
       query: ExportedQuery<import('../app/root').SearchParams>,
     },
     "/:lang?/about": {
-      params: {lang?: string | number},
+      params: {'lang?': string | number},
```

Comment:
Hey there 👋 
I just upgraded the library from `1.0.3` to `1.0.8` and noticed that my routes with [optional segments](https://reactrouter.com/start/library/routing#optional-segments) no longer work.

I think it's due to this specific change because `lang?` means `string | number | undefined` while the new version expects `string | number` only.

Should I open a new issue for this?

Reply:
https://github.com/yesmeck/safe-routes/issues/137

Reply:
Thanks!